deviceaccess.py

```diff
@@ -1049,7 +1088,7 @@ def getTwoDRegisterAccessor(
         Parameters
         ----------
         userType : type or numpy.dtype
-          The userType for the accessor. Can be float, or any of the numpy.dtype
+          The userType for the accessor. Can be of type float, or any of the numpy.dtype
```

Comment:
no, the plain python type float is no longer possible, you replaced it with np.single...

Reply:
Good catch